website/settings/defaults.py

```diff
@@ -227,7 +227,9 @@ def parent_dir(path):
 ALL_MY_REGISTRATIONS_ID = '-amr'
 ALL_MY_PROJECTS_NAME = 'All my projects'
 ALL_MY_REGISTRATIONS_NAME = 'All my registrations'
-NOTEWORTHY_LINKS_NODE = None
+POPULAR_LINKS_NODE = '8w6qe'  # Temporary.  Needs to be set to public node in production.
+NEW_AND_NOTEWORTHY_LINKS_NODE = 'su3q5' # Temporary. Needs to be set to public node in production.
+QA_USER_IDS = ['nxygz', 'x952z', 'tgak8', 'gaexu', 'rgc49', 'nsx26', 'j52af', 'rbk3c', 'xyubm', 'bje5z', 'twkqb', 'mvzr6', 'dihba'] # Should be updated periodically
```

Comment:
What's the process for editing defaults? We have to edit `defaults.py` and commit it to Git again, right? Mixing configuration values into code in this way is generally considered to be bad practice; check out http://12factor.net/config for current best practice. Cleanly separating configuration and code makes it easier to move code through multiple configuration environments (development, staging, production).

I see a [number](https://github.com/CenterForOpenScience/osf.io/blob/17c42034137ca01557163f9d5386450303b872d7/website/settings/defaults.py#L76) of [comments](https://github.com/CenterForOpenScience/osf.io/blob/17c42034137ca01557163f9d5386450303b872d7/website/settings/defaults.py#L119) in defaults.py that suggest that we do use `local.py` to configure our production environment. We should take configuration values out of `defaults.py`, requiring that they be set in `local.py` in both development and production environments.